### PR TITLE
1.9.3.rebased

### PR DESCRIPTION
**Pull request by talbaneth**
1.9.2 was squashed.
I was able to rebase on top of it but can not push to 1.9.3 since it has the old unsquashed 1.9.2 work.
Hence creating this new PR instead of the old 1.9.3 one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,6 +92,7 @@ <h2>Release Calendar</h2>
 				</td>
 				<td>
 					<ul>
+						<li><a href="/releases/kovan/1.9.3/index.html">Release 1.9.3</a> (Wednesday, 2021-04-08)</li>
```
Comment (kmbarry1) [inline index.html:95]: Date seems wrong...maybe it should be `2021-08-04` instead?

Comment (talbaneth) [reply index.html]: Fixed.

File: releases/mainnet/1.9.3/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.3</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.3/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmP1HLPtmE2C8WNpwEXAyFi1LVc3PnT4QLmyCNJxUSCBKj">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.3/abi/index.html">ABIs</a></li>
+				<!--
```
Comment (kmbarry1) [inline releases/mainnet/1.9.3/index.html:15]: @gbalabasquer we will not have a tag for this release either so having this commented out is correct, right?

**Review by kmbarry1 — DISMISSED** [review]
LGTM modulo @gbalabasquer checking the thing I mentioned

**Review by zdumitru — CHANGES_REQUESTED** [review]
Can you please change "active" link for the Kovan release from 1.9.2 to 1.9.3?

**Comment by talbaneth** [comment]
> Can you please change "active" link for the Kovan release from 1.9.2 to 1.9.3?

thanks, done @zdumitru

**Review by zdumitru — APPROVED** [review]

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**